platoon_strategic_IHP/include/platoon_manager_ihp.hpp

```diff
@@ -0,0 +1,265 @@
+
```

Comment:
We have a .h file and a .hpp file with the same base name "platoon_manager_ihp", and a cursory look says they have possibly identical content.  Please choose which naming convention you will use and name all header files with the same extension, and remove whichever is appropriate.  I will review this one; if there are diffs between the two, we will have to worry about them on a second pass.